runtime/onert/core/src/loader/BaseLoader.h

```diff
@@ -1270,6 +1276,35 @@ void BaseLoader<LoaderDomain>::loadFusedBatchNorm(const Operator *op, ir::Graph
   }
 }
 
+template <typename LoaderDomain>
+void BaseLoader<LoaderDomain>::loadBCQUnembedding(const Operator *op, ir::Graph &subg)
```

Comment:
@llFreetimell If BCQ_ is op only for circle, then it should be handled by CircleLoader.